Update AUTHORS file and .mailmap

PatrickJS opened the pull request:
added `LC_ALL=C` due to osx
http://www.inmotionhosting.com/support/website/ssh/speed-up-grep-searche
s-with-lc-all

`Signed-off-by: Patrick Stapleton <github@gdi2290.com>`

jfrazelle commented:
Can you please sign your commits following these rules:

https://github.com/docker/docker/blob/master/CONTRIBUTING.md#sign-your-work

The easiest way to do this is to the last commit:

`git commit --amend -s --no-edit && git push -f`

gdi2290 commented:
@jfrazelle thanks for your help, I was able to sign my commit using my real name

crosbymichael commented:
LGTM

ping @tianon

tianon commented on AUTHORS:
```diff
@@ -306,49 +328,54 @@ Jason Plum <jplum@devonit.com>
+Jean-Tiare Le Bigot <admin@jtlebi.fr>
+Jean-Tiare Le Bigot <jt@yadutaf.fr>
+Jeff Anderson <jeff@docker.com>
+Jeff Anderson <jefferya@programmerq.net>
```
We need to update `.mailmap` to fix some of these duplicates.

gdi2290 replied:
I can updated the .mailmap in this PR

tianon commented on project/generate-authors.sh:
```diff
@@ -11,5 +11,5 @@ cd "$(dirname "$(readlink -f "$BASH_SOURCE")")/.."
 	EOH
 	echo
-	git log --format='%aN <%aE>' | sort -uf
+	git log --format='%aN <%aE>' | LC_ALL=C sort -uf
```
So this is why we've got a lot of the names moving around here?  Does it work more consistently if we use `C.UTF-8`?

gdi2290 replied:
I ran the script again with `C.UTF-8` which seems to work fine. The names were moved around due to the locale change

```bash
$ LC_ALL=en_US sort <<< $'a\nb\nA\nB'
a
A
b
B
$ LC_ALL=C sort <<< $'a\nb\nA\nB'
A
B
a
b
```

icecrime commented on AUTHORS:
```diff
@@ -598,12 +645,13 @@ shaunol <shaunol@gmail.com>
 Shawn Siefkas <shawn.siefkas@meredith.com>
 Shih-Yuan Lee <fourdollars@gmail.com>
+Shishir Mahajan <shishir.mahajan@redhat.com>
+shishir-a412ed <smahajan@redhat.com>
```
Another dupe!

gdi2290 replied:
thanks I updated the PR

gdi2290 commented on AUTHORS:
```diff
@@ -109,6 +114,7 @@ Caleb Spare <cespare@gmail.com>
 Calen Pennington <cale@edx.org>
 Cameron Boehmer <cameron.boehmer@gmail.com>
 Carl X. Su <bcbcarl@gmail.com>
+cc272309126 <cc272309126@gmail.com>
```
I found this guy via google search as Chen Chao. I'm assuming I should update .mailmap with his correct name

gdi2290 replied:
actually I found him here https://github.com/docker/docker/commit/1bb02117db80e75f406f6c63d8d50680c1569019
I'm updating the `.mailmap` again

gdi2290 commented on AUTHORS:
```diff
@@ -132,15 +140,22 @@ Colin Rice <colin@daedrum.net>
 Daan van Berkel <daan.v.berkel.1980@gmail.com>
+daehyeok mun <daehyeok@daehyeokui-MacBook-Air.local>
+Daehyeok Mun <daehyeok@gmail.com>
 Daehyeok.Mun <daehyeok@gmail.com>
```
lol `daehyeokui-MacBook-Air.local` seems legit. I'll flatten these 3 in `.mailmap`

gdi2290 commented on AUTHORS:
```diff
@@ -197,23 +212,26 @@ Eike Herzbach <eike@herzbach.net>
 Evan Krall <krall@yelp.com>
 Evan Phoenix <evan@fallingsnow.net>
 Evan Wies <evan@neomantra.net>
+evanderkoogh <info@erronis.nl>
```
this guy didn't sign off his commit so I'll assume he is fine with the name on his github profile

gdi2290 commented on AUTHORS:
```diff
@@ -367,21 +395,22 @@ Kawsar Saiyeed <kawsar.saiyeed@projiris.com>
 kies <lleelm@gmail.com>
-kim0 <email.ahmedkamal@googlemail.com>
 Kim BKC Carlbacker <kim.carlbacker@gmail.com>
+kim0 <email.ahmedkamal@googlemail.com>
```
another person who didn't sign their commits so again I'll use his github name

gdi2290 commented on AUTHORS:
```diff
@@ -412,12 +444,13 @@ Manuel Meurer <manuel@krautcomputing.com>
 Marcus Farkas <toothlessgear@finitebox.com>
-marcuslinke <marcus.linke@gmx.de>
 Marcus Ramberg <marcus@nordaaker.com>
+marcuslinke <marcus.linke@gmx.de>
```
again didn't sign commit 
https://github.com/docker/docker/commit/520163a00ee22ddb3d689f7304db2af2784d3bdb
I'll assume his name is `Marcus Linke` from his email/username

gdi2290 commented on AUTHORS:
```diff
@@ -491,18 +531,21 @@ NikolaMandic <mn080202@gmail.com>
 Olivier Gambier <dmp42@users.noreply.github.com>
-O.S. Tezer <ostezer@gmail.com>
 pandrew <letters@paulnotcom.se>
+panticz <mail@konczalski.de>
```
these commits were signed off by another person `Sven Dowideit` so not sure what to do here
https://github.com/docker/docker/commits?author=panticz

gdi2290 commented on AUTHORS:
```diff
@@ -518,16 +560,18 @@ Peter Braden <peterbraden@peterbraden.co.uk>
+Pierre-Alain RIVIERE <pariviere@ippon.fr>
 Piotr Bogdan <ppbogdan@gmail.com>
 pixelistik <pixelistik@users.noreply.github.com>
+Porjo <porjo38@yahoo.com.au>
```
`Ian Bishop <ianbishop@pace7.com>` I'm pretty certain these are the same since google searching shows other projects being sign by `Ian B <porjo38@yahoo.com.au>`
https://github.com/docker/docker/commits?author=Porjo

gdi2290 commented on AUTHORS:
```diff
@@ -518,16 +560,18 @@ Peter Braden <peterbraden@peterbraden.co.uk>
+Porjo <porjo38@yahoo.com.au>
 Prasanna Gautam <prasannagautam@gmail.com>
 Przemek Hejman <przemyslaw.hejman@gmail.com>
 pysqz <randomq@126.com>
```
this guy didn't sign his commits
https://github.com/docker/docker/commits?author=pysqz

gdi2290 commented on AUTHORS:
```diff
@@ -547,6 +591,7 @@ Renato Riccieri Santos Zannon <renato.riccieri@gmail.com>
 rgstephens <greg@udon.org>
```
another unsigned
https://github.com/docker/docker/commits?author=rgstephens

gdi2290 commented on AUTHORS:
```diff
@@ -619,8 +667,12 @@ Steven Merrill <steven.merrill@gmail.com>
 sudosurootdev <sudosurootdev@gmail.com>
```
not signed
https://github.com/docker/docker/commit/2a1181f404387f6d530a86a6196b40a6dd541c20

gdi2290 commented on AUTHORS:
```diff
@@ -619,8 +667,12 @@ Steven Merrill <steven.merrill@gmail.com>
+Sébastien <sebastien@yoozio.com>
+Sébastien Luttringer <seblu@seblu.net>
+Sébastien Stormacq <sebsto@users.noreply.github.com>
 tang0th <tang0th@gmx.com>
```
not signed off
https://github.com/docker/docker/commits?author=tang0th

PatrickJS committed:
Update AUTHORS file and .mailmap

added `LC_ALL=C.UTF-8` due to osx
http://www.inmotionhosting.com/support/website/ssh/speed-up-grep-searche
s-with-lc-all

Signed-off-by: Patrick Stapleton <github@gdi2290.com>

gdi2290 commented:
@tianon any more suggestions for this PR? 
most the of people who aren't signed off are fine since they're small PRs

tianon commented:
Naw seems good enough.  Thanks for sticking with it!

LGTM

crosbymichael commented:
LGTM
